src/workflows/airqo_etl_utils/config.py

```diff
@@ -205,6 +205,12 @@ class Config:
     TWITTER_BOT_ACCESS_TOKEN = os.getenv("TWITTER_BOT_ACCESS_TOKEN")
     TWITTER_BOT_ACCESS_TOKEN_SECRET = os.getenv("TWITTER_BOT_ACCESS_TOKEN_SECRET")
 
+    #Pull data from satellite
+    BIGQUERY_SITES = os.getenv("BIGQUERY_SITES")
+    GOOGLE_APPLICATION_CREDENTIALS_EMAIL = os.getenv("GOOGLE_APPLICATION_CREDENTIALS_EMAIL")
+    MONGODB_URI = os.getenv("MONGODB_URI")
+    MONGODB_DATABASE = os.getenv("MONGODB_DATABASE")
+    MONGODB_COLLECTION = os.getenv("MONGODB_COLLECTION")
```

Comment:
hello @wabinyai, some of these secrets already exist e.g MONGOURI, so no need for new uris maybe DBs, amd  perhaps a new collection i know MONGO_COLLECTION already exists, so perhaps use SATELITE_MONGO_COLLECTION . Skim through the list of existing ones in this very config file
```suggestion
    BIGQUERY_SITES = os.getenv("BIGQUERY_SITES")
    GOOGLE_APPLICATION_CREDENTIALS_EMAIL = os.getenv("GOOGLE_APPLICATION_CREDENTIALS_EMAIL")
    MONGODB_URI = os.getenv("MONGODB_URI")
    MONGODB_DATABASE = os.getenv("MONGODB_DATABASE")
    SATELITE_DATA_COLLECTION = os.getenv("SATELITE_DATA_COLLECTION")
```